packages/svelte/src/internal/client/dom/elements/events.js

```diff
@@ -28,6 +54,10 @@ export function event(event_name, dom, handler, capture, passive) {
 
 	dom.addEventListener(event_name, target_handler, options);
 
+	if (hydrating) {
+		hydrate_event_replay(/** @type {HTMLElement} */ (dom));
+	}
```

Comment:
why are we adding this here? as @dummdidumm noted, this means we include this code in almost every app. I'd expect something more like this...

```diff
import * as $ from "svelte/internal/client";

var root = $.template(`<img alt="blah" src="blah.jpg">`);

export default function App($$anchor) {
  function onload() {
    console.log('loaded');
  }

  var img = root();

+ $.reload(img);
  $.event("load", img, onload, false);
  $.append($$anchor, img);
}
```

...where that line is only added for relevant elements (including `<svelte:element>` i guess) with a spread or an `onload` or an `onerror` (plus _maybe_ actions, depending on what we decide)

Reply:
(maybe `$.reload` is a misleading name but you get the idea)